docs/development/SETUP.md

```diff
@@ -70,6 +70,24 @@ VEDA-UI includes a GitHub action for checking TypeScript and lint errors. If you
 
 Additionally, there's a pre-commit hook that performs the same error checks. This helps developers identify and address issues at an earlier stage. If you need to bypass the check (to make a local temporary commit etc.), include the `--no-verify`` flag in your commit command.
 
+#### Format on Save
+While the existing checks ensure that no linting errors are introduced into the main branch, enabling "Format on Save" in your code editor provides an additional layer of convenience. It helps catch formatting issues early in the development process, minimizing the risk of committing errors and ensuring consistent code style across the team.
+
+##### How to Set Up Format on Save in VS Code
+Install Required Extensions:
+
+1. Install the ESLint extension ("dbaeumer.vscode-eslint").
+2. Enable Format on Save in the settings (<kbd>⌘ Command</kbd> + <kbd>,</kbd> on macOS)
+3. Configure Formatter Settings:
+   Ensure ESLint is set as the default formatter:
+   ```json
+   "editor.defaultFormatter": "dbaeumer.vscode-eslint"
+   ```
```

Comment:
was able to sortof follow.. following step 2, couldn't figure out where to add snippet so just opened command palette with cmd+shift+p and found `Open User Settings (json)` and then added snippet.

Also must be a small type but i dont think `json` is meant to be after the "```"